docs-release-notes.md

```diff
@@ -1,3 +1,20 @@
+# July 26, 2024
+
+## Device Sync
+
+- Get Started: Rename references to "Realm" and adjust phrasing for clarity.
+- Configure Sync/Sync Settings: Update page to indicate that the ``Sync Type`` field only displays for Partition-Based Sync apps and move ``Select Your Cluster`` field to the top of the page.
```

Comment:
```suggestion
- Configure Sync/Sync Settings: Update page to indicate that the Sync Type field only displays for Partition-Based Sync apps and move the "Cluster to Sync" section to the top of the page.
```
Monospace should only be used for variable names. We could use :guilabel:, which is used to denote UI components, but since this is internal and won't be posted on the site, think it's fine to leave as regular text. I also see that we've never used :guilabel: in release notes before, so we'll remain consistent. 

Looked at the PR, and I think the section name was supposed to be "Cluster to Sync". Section names should be in quotes.